lib/internal/Magento/Framework/Exception/AggregateExceptionInterface.php

```diff
@@ -0,0 +1,21 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento\Framework\Exception;
+
+/**
+ * Exception with possibility to set several error messages
+ *
+ * @api
+ */
+interface AggregateExceptionInterface
+{
+    /**
+     * Get the array of LocalizedException objects. Get an empty array if no errors were added
+     *
+     * @return LocalizedException[]
```

Comment:
getErrors returns `LocalizedException[]` array to be compatible with current Implementation in Web API which relies on this behavior.

`@see` the \Magento\Framework\Webapi\Exception 
which receives $errors as a set of Localized Exceptions
 ```* @param \Magento\Framework\Exception\LocalizedException[]|null $errors Array of errors messages```